src/Apps/Artwork/Components/__stories__/ArtistInfo.story.tsx

```diff
@@ -1,27 +1,21 @@
 import { Box } from "@artsy/palette"
-import { bio, biography_blurb } from "Apps/__test__/Fixtures/ArtistBio"
-import { artistResponse } from "Apps/__test__/Fixtures/MarketInsights"
-import { exhibitions } from "Apps/__test__/Fixtures/SelectedExhibitions"
+// import { bio, biography_blurb } from "Apps/__test__/Fixtures/ArtistBio"
+// import { artistResponse } from "Apps/__test__/Fixtures/MarketInsights"
+// import { exhibitions } from "Apps/__test__/Fixtures/SelectedExhibitions"
 import React from "react"
 import { storiesOf } from "storybook/storiesOf"
 import { Section } from "Styleguide/Utils/Section"
 import { RelayStubProvider } from "Utils/RelayStubProvider"
-import { ArtistInfo } from "../ArtistInfo"
+import { ArtistInfoQueryRenderer } from "../ArtistInfo"
 
 storiesOf("Styleguide/Artwork", module)
   .addDecorator(story => <RelayStubProvider>{story()}</RelayStubProvider>)
   .add("ArtistInfo", () => {
     return (
       <React.Fragment>
-        <Section title="Artist Info">
+        <Section title="ArtistInfoQueryRenderer">
           <Box width="100%">
-            <ArtistInfo
-              name="Francesca DiMattio"
-              artist={artistResponse as any}
-              exhibitions={exhibitions as any}
-              bio={bio}
-              credit={biography_blurb.credit as any}
-            />
+            <ArtistInfoQueryRenderer artistID="pablo-picasso" />
```

Comment:
Does this have to be inside a box?

Reply:
This is so the width of each sub component in ArtistInfo extends out to 100% of the viewport size; otherwise each of the composing components have their own max-widths, leading to staggering. I wasn't totally sure what would happen if I updated each little sub component inside of here to fill the screen so this was safest. Thoughts?

Reply:
Oh. Makes sense. I think it is logical for parent to deal with the width.